token/program-2022/src/extension/mod.rs

```diff
@@ -670,12 +670,15 @@ pub enum ExtensionType {
     ConfidentialTransferFeeAmount,
     /// Mint contains a pointer to another account (or the same account) that holds metadata
     MetadataPointer,
+    /// Test unsized mint extension
+    #[cfg(test)]
+    UnsizedMintTest = u16::MAX - 2,
     /// Padding extension used to make an account exactly Multisig::LEN, used for testing
     #[cfg(test)]
-    AccountPaddingTest = u16::MAX - 1,
+    AccountPaddingTest,
     /// Padding extension used to make a mint exactly Multisig::LEN, used for testing
     #[cfg(test)]
-    MintPaddingTest = u16::MAX,
+    MintPaddingTest,
```

Comment:
Is there a reason to remove the variant index assignments that I missed?

Reply:
just to avoid the verbosity of setting at each one, and instead just setting it at the lowest test one, trusting the enum to assign the correct numbers. Should I revert?

Reply:
It's probably fine; I don't feel strongly. Feel free to merge as-is :)